pkg/mcs/resource_manager/client/client.go

```diff
@@ -351,6 +352,9 @@ type groupCostController struct {
 		consumption *rmpb.Consumption
 	}
 
+	// fast path to make once token limit with un-limit burst.
+	burstable *atomic.Bool
```

Comment:
It seems that the standard library has an `atomic.Bool` type, can we use that one?

https://pkg.go.dev/sync/atomic#Bool

Reply:
addressed